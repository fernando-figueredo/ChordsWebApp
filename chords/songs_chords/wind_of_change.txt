f f f f f f f dm d dm dm dm dm dm dm f f f f f f f dm dm dm dm dm am dm dm e am am am am am dm dm dm dm dm am am am g g g g g g g g g g g g bm em 