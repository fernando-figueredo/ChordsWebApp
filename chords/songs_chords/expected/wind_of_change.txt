f f f f f f f dm dm dm dm dm dm dm f f f f f f f dm dm dm dm dm dm dm am am am am am am am dm dm dm am am am g g g g g g g